tasks/src/visual-question-answering/data.ts

```diff
@@ -0,0 +1,59 @@
+import { type TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [],
+	demo:     {
+		inputs:  [
+			{
+				filename: "elephant.jpeg",
+				type:     "img",
+			},
+			{
+				label:   "Question",
+				content: "Is there an elephant?",
+				type:    "text",
+			},
+		],
+		outputs: [
+			{
+				type: "chart",
+				data: [
+					{
+						label: "yes",
+						score: 0.999,
+					},
+					{
+						label: "no",
+						score: 0.009,
+					},
+					{
+						label: "1",
```

Comment:
The probabilities don't have to add up to 1 since they are not mutually exclusive for more complex questions, such as "what is on top of ...". 

Here's the output of the specific model: 
```python
>>> image =  Image.open("assets/elephant.jpeg")
>>> question = "Is there an elephant?"
>>> vqa_pipeline(image, question, top_k=3)
[{'score': 0.9998154044151306, 'answer': 'yes'}, 
 {'score': 0.009802112355828285, 'answer': 'no'}, 
 {'score': 0.0020384755916893482, 'answer': '1'}]
>>> 
```
Do you think it would make it more clear if we increase the number of outputs `top_k` to make the sum go way over 1 so this is not confusing?

I thought about removing the "1" label too, but decided to keep it because it kind of showed that VQA's outputs are not just binary. Maybe we could show this better by asking a different question, such as "where is the elephant?".

Reply:
@jlondonob sure, you can also include a better input not asking for a yes/no question but something more like "What's in this image?" Thanks a lot for the explanation!